docs/sources/dashboards/share-dashboards-panels/index.md

```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
 As of Grafana 8.0, anonymous access permission is not available in Grafana Cloud.
 {{< /admonition >}}
 
-When you share a panel or dashboard as a snapshot, a snapshot (which is a panel or dashboard at the moment you take the snapshot) is publicly available on the web. Anyone with a link to it can access it. Because snapshots do not require any authorization to view, Grafana removes information related to the account it came from, as well as any sensitive data from the snapshot.
-
 ## Share a dashboard
 
-You can share a dashboard as a direct link or as a snapshot. You can also export a dashboard.
+You can share dashboards in the following ways:
 
-{{< admonition type="note" >}}
-If you change a dashboard, ensure that you save the changes before sharing.
-{{< /admonition >}}
+- Internally with a link
+- Externally with a link (to anyone)
+- Externally by email (to specific people)
+- As a snapshot
+- As a report
+- As a JSON file
+
```

Comment:
Should we include "As a PDF" as well? The changes are in review for that

Reply:
Will this be behind the newDashboardShareComponent feature flag as well?